airbyte-ci/connectors/pipelines/pipelines/gradle.py

```diff
@@ -60,26 +70,35 @@ async def _get_patched_build_src_dir(self) -> Directory:
         )
         return build_src_dir.with_new_file("src/main/groovy/airbyte-connector-acceptance-test.gradle", contents=cat_gradle_plugin_content)
 
-    def _get_gradle_command(self, extra_options: Tuple[str] = ("--no-daemon", "--scan", "--build-cache")) -> List:
+    def _get_gradle_command(self, extra_options: Tuple[str, ...] = ("--no-daemon", "--scan", "--build-cache")) -> List:
         command = (
             ["./gradlew"]
             + list(extra_options)
             + [f":airbyte-integrations:connectors:{self.context.connector.technical_name}:{self.gradle_task_name}"]
+            + list(self.gradle_task_options)
         )
         for task in self.DEFAULT_TASKS_TO_EXCLUDE:
             command += ["-x", task]
         return command
 
     async def _run(self) -> StepResult:
+        includes = self.build_include
+        if self.with_java_cdk_snapshot:
+            includes + ["./airbyte-cdk/java/airbyte-cdk/**"]
```

Comment:
I tried that upstream method first, but for some reason the upstream reference was not sufficient. It may be to do with how globs are processed, or some other niche issue. I've logged for follow-up:

- https://github.com/airbytehq/airbyte/issues/29667